errors/errors.go

```diff
@@ -30,6 +30,7 @@ const (
 	DNS
 	BadPublicKey
 	BadCSR
+	AlreadyRevoked
```

Comment:
Should I actually be adding this at the berrors level, or should the code which returns this error actually just be directly returning a `probs.AlreadyRevoked` instead?

Reply:
As per discussion in our meeting just now, this is the preferred way to do this. The WFE can convert a `berrors.AlreadyRevoked` into a `probs.AlreadyRevoked` using the `web.problemDetailsForBoulderError` converter.